eo-maven-plugin/src/main/java/org/eolang/maven/OptimizeMojo.java

```diff
@@ -273,7 +273,10 @@ private Path make(final XML xml, final Path file) throws IOException {
         final Path target = place.make(
             this.targetDir.toPath().resolve(OptimizeMojo.DIR), TranspileMojo.EXT
         );
-        new Home().save(xml.toString(), target);
+        new Home(this.targetDir.toPath()).save(
+            xml.toString(),
+            this.targetDir.toPath().relativize(target)
```

Comment:
In this case we can use something like:
```jar
        new Home(targetDir.toPath()).save(
            xml.toString(),
            place.make(Paths.get(OptimizeMojo.DIR), TranspileMojo.EXT)
        );
```
and probably we will able to remove `target` variable at all (I can be wrong there). But it doesn't metter. I'm about slightly different: 
as I see in most cases the `relativize` method is used as a hack to make the pluging and tests work. Maybe we need a separate pazzle or an issue in order to remove all `relativize` invocations and to make the code cleaner and more straightforward. wdyt?

Reply:
@volodya-lombrozo `place.make()` creates an absolute path and we need relative here.
You are correct: `relativize` is used to adapt absolute paths used in client code to relative ones required by `Home`. Initially I was going to refactor the code in clients so that `relativize` is not used, but it proved to be very expensive refactoring and needs to be considered for each client class individually.
I believe single issue to correct them all would be inappropriately big task.

Reply:
Ok